packages/devtools_server/lib/src/server.dart

```diff
@@ -217,7 +217,7 @@ Future<void> registerLaunchDevToolsService(
         // quietly replace the IP with "penguin.linux.test". This is not valid
         // for us since the server isn't bound to the containers IP (it's bound
         // to the containers loopback IP).
-        path: devToolsUri.path == '' ? '/' : devToolsUri.path,
+        path: devToolsUri.path.isEmpty ? '/' : devToolsUri.path,
```

Comment:
I suspect that there's a lint for this...

Reply:
I found one for collections (https://dart-lang.github.io/linter/lints/prefer_is_empty.html), but not strings :(